README.md

```diff
@@ -0,0 +1,135 @@
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder: 
+
+
+ Commands:
+
+```sh
+  cd hello-microverse-project
+  https://github.com/rodrigopenayo98/hello-microverse-project.git
+```
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Rodrigo Penayo**
+
+- GitHub: [@rodrigopenayo98](https://github.com/rodrigopenayo98)
+- Twitter: [@rodrigopenayo98](https://twitter.com/rodrigopenayo98)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rodrigo-penayo-391226158/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
- [ ] Good job with this, @rodrigopenayo98 :smile: but I think you can make it more custom and applicable to your project rather than leaving the template unchanged. This allows you to show others that you are aware of what can be improved on, or how to enhance the present version of your project. Even though `Hello Microverse` doesn't have much to be done with as it is just a preparation for the program. For example, you might choose to say that you intend to add the following features in the future:
  - Responsiveness and adaptability to all devices
  -  Add more pages
  - Introduce more CSS styles

_I hope you get the logic now :wink:_